modules/appeals_api/spec/models/higher_level_review_spec.rb

```diff
@@ -183,15 +183,15 @@
     end
     let(:api_version) { 'V1' }
 
-    context 'birth date isn\'t a date' do
-      let(:auth_headers) { default_auth_headers.merge 'X-VA-Birth-Date' => 'apricot' }
-
-      it 'using a birth date that isn\'t a date creates an invalid record' do
-        expect(higher_level_review.valid?).to be false
-        expect(higher_level_review.errors.to_a.length).to eq 1
-        expect(higher_level_review.errors.to_a.first.downcase).to include 'isn\'t a date'
-      end
-    end
+    # context 'birth date isn\'t a date' do
```

Comment:
Go ahead and permanently remove all tests related to the removed model validations.